.github/styles/vocab.txt

```diff
@@ -215,6 +216,7 @@ prebaked
 preconfigured
 prepack
 productional
+programatically
```

Comment:
Not sure about this one https://en.m.wiktionary.org/wiki/programatically 😅

Reply:
😅oof